src/app/filter-gender/filter-gender.component.html

```diff
@@ -0,0 +1,10 @@
+<h3>For who are you packing?</h3>
+
+<button mat-button [ngClass]="{selected: male}" (click)="maleSelect()">
+    <i class="material-icons">person</i>
+    <p>Male</p>
+</button>
+<button mat-button [ngClass]="{selected: female}" (click)="femaleSelect()">
```

Comment:
There's an alternative way to pass the conditional class - `[class.selected]="female"` Pls re-check.